Wayfindr Demo/Classes/Controller/Maintainer/BeaconsInRangeViewController.swift

```diff
@@ -25,6 +25,30 @@
 
 import UIKit
 import CoreLocation
+// FIXME: comparison operators with optionals were removed from the Swift Standard Libary.
```

Comment:
Same as above about referencing a GitHub issue. Should all probably be the same issue, as it's all comparing optionals.